CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
 Changelog for Mantarray Frontend Components
 ===========================================
+0.1.9 (2020-12-22)
```

Comment:
until we actually publish a new version, please just put `0.1.9 (unreleased)`

Reply:
Modified.